grouper/fe/templates/service.html

```diff
@@ -26,7 +26,8 @@
     <!-- Enable/Disable button. -->
```

Comment:
Nit: could we rename this file to `role-user.html` while we're here?

Reply:
Good idea. Done.